File: push-xiaomi/src/main/java/net/coding/program/push/xiaomi/PushAction.java
package net.coding.program.push.xiaomi;

import android.content.Context;

/**
 * Created by chenchao on 2017/11/2.
 */

public interface PushAction {

    String TAG = "CodingPush";

    boolean init(Context context, CommonPushClick ClickPushActionName);

    void bindGK(Context context, String gk);

    void unbindGK(Context context, String gk);

}

File: push-xiaomi/src/main/java/net/coding/program/push/xiaomi/PushAction.kt
package net.coding.program.push.xiaomi

import android.content.Context

/**
 * Created by chenchao on 2017/11/2.
 */

interface PushAction {

    fun init(context: Context, ClickPushActionName: CommonPushClick): Boolean

    fun bindGK(context: Context, gk: String)

    fun unbindGK(context: Context, gk: String)

    companion object {

        val TAG = "CodingPush"
    }

}
